libs/execution_context/ebpf_core.c

```diff
@@ -2707,6 +2707,7 @@ typedef struct _ebpf_protocol_handler
 #if !defined(CONFIG_BPF_INTERPRETER_DISABLED)
             uint64_t used_by_interpret : 1;
 #endif
+            uint64_t privileged : 1; // If set, the operation can only be called by the eBPF Service.
```

Comment:
How about extending `_ebpf_operation_header` with an optional SDDL?

Reply:
Unfortunately, SDDL parsing support doesn't exist in the kernel. SDDL to SD conversion is a user mode only construct.,